pkg/virt-controller/services/template.go

```diff
@@ -22,7 +22,9 @@ package services
 import (
 	"context"
 	"fmt"
+	operatorutil "kubevirt.io/kubevirt/pkg/virt-operator/util"
```

Comment:
nit: can we move this to the group with local imports?

Reply:
Ack. Done.